packages/contracts-ops/contracts/Config.sol

```diff
@@ -18,6 +18,7 @@ import {INomadProtocol} from "./test/utils/INomadProtocol.sol";
 
 import "forge-std/Vm.sol";
 import "forge-std/Test.sol";
+import "forge-std/console2.sol";
```

Comment:
leftover?

Reply:
@odyslam why convo resolved - shouldn't this be removed?

Reply:
Yes, mistook the console2 import for the main script